### ISSUE TITLE
Adding more maintainers to help keep the code working with new python versions 

### ISSUE THREAD

**Issue by sanjayankur31** [aarongarrett/inspyred#29]
Hi @aarongarrett , I was wondering what if you thought of giving a few contributors access to the repository and pypi so that we can help with at least merging PRs and keeping the code working with newer python releases, and then being able to release these on pypi?

I'm happy to help with general maintenance---we use inspyred in our tools so and we'd like to keep using it on newer python versions. I'm also happy to help with pypi releases (we can just set up a github  action that automatically releases to pypi when a release is tagged here on github). 

https://packaging.python.org/en/latest/guides/publishing-package-distribution-releases-using-github-actions-ci-cd-workflows/

Please do let us know. We'd really like to keep inspyred alive. Cheers,

**Comment by aarongarrett** [comment]
Yes. Anyone who wants to be granted access can just let me know. I would
love that.

--
Aaron Garrett

**Comment by sanjayankur31** [comment]
That's great to hear. I'm happy to help out to begin with. I'd like to add more CI to cover all current supported Python versions, fix any issues we see, and then make a new release with these fixes so folks can use inspyred off pypi again.

**Comment by sanjayankur31** [comment]
I've released 1.0.2 with a bunch of fixes, please do test it out and file new issues if you encounter them

https://pypi.org/project/inspyred/1.0.2/

Leaving this issue open for others.
